### PR TITLE
Spg 219 adding release repo config 

### PR DESCRIPTION
**Pull request by RajBaviskar**
- Adding yaml field.

### CODE REVIEW

File: internal/service/platform/service/data_source_service.go [hunk 1]
```diff
@@ -19,7 +19,13 @@ func DataSourceService() *schema.Resource {
 
 		ReadContext: dataSourceServiceRead,
 
-		Schema: map[string]*schema.Schema{},
+		Schema: map[string]*schema.Schema{
+			"yaml": {
+				Description: "Input Set YAML",
+				Type:        schema.TypeString,
+				Optional:    true,
```
Comment (rathodmeetsatish) [inline internal/service/platform/service/data_source_service.go:26]: This should not be optional true this should be computed true

File: internal/service/platform/service/data_source_service_test.go [hunk 2]
```diff
@@ -103,6 +108,11 @@ func testAccDataSourceServiceByName(id string, name string) string {
 			name = "%[2]s"
 			org_id = harness_platform_project.test.org_id
 			project_id = harness_platform_project.test.id
+			yaml = <<-EOT
```
Comment (rathodmeetsatish) [inline internal/service/platform/service/data_source_service_test.go:111]: yaml should not be provided in data source

File: internal/service/platform/service/data_source_service.go (hunk 1 above)
Comment (RajBaviskar) [reply internal/service/platform/service/data_source_service.go]: fixed.

File: internal/service/platform/service/data_source_service_test.go (hunk 2 above)
Comment (RajBaviskar) [reply internal/service/platform/service/data_source_service_test.go:106]: removed.

**Review by rijajoo — APPROVED** [review]

**Review by rathodmeetsatish — APPROVED** [review]

**Pull request merged**